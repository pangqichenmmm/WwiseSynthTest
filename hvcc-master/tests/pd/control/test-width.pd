#N canvas 447 194 167 93 10;
#X obj 15 10 loadbang, f 13;
#X obj 15 56 print, f 10;
#X msg 15 33 hello, f 9;
#X connect 0 0 2 0;
#X connect 2 0 1 0;
