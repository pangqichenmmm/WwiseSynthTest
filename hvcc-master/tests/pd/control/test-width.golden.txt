[@ 0.000] print: hello
